src/main/java/com/iota/iri/service/TipsSolidifier.java

```diff
@@ -30,9 +30,15 @@ public TipsSolidifier(final Tangle tangle,
     public void init() {
         solidityRescanHandle = new Thread(() -> {
 
+            long lastTime = 0;
             while (!shuttingDown) {
                 try {
                     scanTipsForSolidity();
+                    long now = System.currentTimeMillis();
```

Comment:
Since you will only run this logic in debug mode and you don't want to do a system call every time, add `log.isDebugEnabled()`.

Reply:
and we're done :-)

Reply:
@GalRogozinski i am not sure why the build [failed](https://travis-ci.org/iotaledger/iri/jobs/415925027)